File: backend/src/main/java/com/simats/neoomfs/dto/response/RadiologyResponse.java
package com.simats.neoomfs.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class RadiologyResponse {
    private Long id;
    private Long patientId;
    private boolean iopaTaken;
    private String iopaFileUrl;
    private String iopaFindings;
    private boolean opgTaken;
    private String opgFileUrl;
    private String opgFindings;
    private boolean cbctTaken;
    private String cbctFileUrl;
    private String cbctFindings;
    private Double boneDensityHu;
    private String generalRadiologyNotes;
}

File: frontend/app/src/main/java/com/simats/neoomfs/models/RadiologyResponse.kt
package com.simats.neoomfs.models

data class RadiologyResponse(
    val id: Long?,
    val patientId: Long?,
    val iopaTaken: Boolean,
    val iopaFileUrl: String?,
    val iopaFindings: String?,
    val opgTaken: Boolean,
    val opgFileUrl: String?,
    val opgFindings: String?,
    val cbctTaken: Boolean,
    val cbctFileUrl: String?,
    val cbctFindings: String?,
    val boneDensityHu: Double?,
    val generalRadiologyNotes: String?
)
